.changeset/sweet-carrots-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sl-design-system/avatar': patch
+---
+
+Due to #1371, the subtitle is no longer shown for the `md` size
```

Comment:
Are we sure that this will not cause a breaking change for people, who are already using subtitle in `md` size?

Reply:
I will highlight this change in Slack to be sure, but i don't consider this breaking enough for a major.